### ISSUE TITLE
Please verify syntax and try again.

### ISSUE THREAD

**Issue by doged** [vindimy/altcointip#40]
after i attempt to tip, it sends me a msg with a syntax issue.. ive pasted the debug.. 

http://pastebin.com/4Cq8BQCX

**Comment by doged** [comment]
btw, everything else works..

**Comment by doged** [comment]
+history, +withdraw, +info, and +register all work, just wont tip, keeps msg'ing a syntax error.. i've attempted almost every variation of tipping i could think of. all i can think is that it maybe the regex somewhere..

**Comment by doged** [comment]
here is a more updated log of whats happening.. really, everything works but tipping. its so strange.
http://pastebin.com/iN5rhuA2

**Comment by doged** [comment]
well, i fixed the traceback py errors, there was a %s missing in line 1099 of ctb_action.py but tip is still saying im not giving it the right syntax.. any ideas?

**Comment by doged** [comment]
also a typo (givetipt instead of givetip)

**Comment by josephdyland** [comment]
I am having this same error, Did you ever fix this?  +history, +withdraw, +info, and +register all work, just wont tip

The bot messages me back with;
Please verify syntax and try again

**Comment by justinvforvendetta** [comment]
yeah i cant get withdraw to work now, but tipping works just fine.

**Comment by justinvforvendetta** [comment]
ill upload my source code in a minute

**Comment by justinvforvendetta** [comment]
hey @josephdyland got an email address? i planned on going over this tonight to fix it, and get it up and running so i can take it off my to do list..

**Comment by justinvforvendetta** [comment]
ok i think the issue i had with tipping was in ctb_action, sendtoaddr needed to be sendtoaddress

**Comment by justinvforvendetta** [comment]
it occurs twice, lines 656, and 662. let me know if that helps your tip issue.

**Comment by justinvforvendetta** [comment]
oh i meant to ask, whats the best way to get this to run in the background without screen? normally with nodejs id use forever, but im not sure how to daemonize this, so to speak.

**Comment by josephdyland** [comment]
@justinvforvendetta , I ended up resolving my issues for now, I had many ..  I now have all features of the bot working,  I am running it in the background with command . nohup sh _start.sh &

**Comment by justinvforvendetta** [comment]
awesome! was the issue i listed above part of it? what was preventing tip? i cant figure out why i can tip but cant withdraw now, i started off with the opposite issue lol

**Comment by doged** [comment]
@josephdyland any idea on why everything but withdraw works?

**Comment by josephdyland** [comment]
Is your coin symbol more then 3 letters?  My install is for a coin with 5 letters. When I set it up in coin.yml I had defined it as a 3 letter coin and the only thing that works for withdraws is using that same 3 letter symbol. eg

If coin.yml looks like

dog:
    enabled: true
    unit: dog
    name: Dogecoin Dark

The Withdraw command would look like this.

+withdraw ADDRESS AMOUNT DOG
